src/lib/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { configure, ConfigureOptions, Environment } from 'nunjucks'
+import { dirname, join } from 'path'
+
+/**
+ * Get the template paths that should be specified when manually creating a new
+ * Nunjucks environment.
+ */
+export function getNunjucksPaths() {
+  const govukFrontendPath = dirname(dirname(require.resolve('govuk-frontend')))
+  return [
+    join(__dirname, '..', 'nunjucks'),
+    govukFrontendPath,
+  ]
+}
+
+type NunjucksFilter = Parameters<Environment['addFilter']>[1]
+
+const addCustomMODUKClass: NunjucksFilter = (params, classToAdd: string, options?: { not?: RegExp }) => {
+  const classes: string[] = (params.classes ?? '').split(' ')
```

Comment:
not?: RegExp could be changed to `RegExp | string`  but the ts types for Nunjunks are mostly anys so there no much benefit